rust/worker/src/execution/orchestration/compact.rs

```diff
@@ -104,15 +116,20 @@ pub struct CompactOrchestrator {
     pulled_log_offset: Option<i64>,
     // Dispatcher
     dispatcher: ComponentHandle<Dispatcher>,
-    // number of write segments tasks
-    num_write_tasks: i32,
+    // Tracks the total remaining number of MaterializeLogs tasks
+    num_uncompleted_materialization_tasks: usize,
+    // Tracks the total remaining number of tasks per segment
```

Comment:
This is write tasks + commit tasks + flush tasks right?

Reply:
yep